db/seeds.rb

```diff
@@ -5,3 +5,14 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+second_user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
+third_user = User.create(name: 'Zain', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Pakistan.')
+fourth_user = User.create(name: 'John', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Canada.')
+
+first_post = Post.create(title: 'Hello1', text: 'This is my first post1', user_id: first_user.id)
+second_post = Post.create(title: 'Hello2', text: 'This is my first post2', user_id: second_user.id)
+
+Comment.create(post: first_post.id, author: first_user.id, text: 'Hi Tom!1')
+Comment.create(post: second_post.id, author: second_user.id, text: 'Hi Tom!2')
```

Comment:
- [ ] I tried seeding your data but it fails. The reason why it fails is because of the way you are creating your `Comment`.  You are making use of `author` column which does not exist in your database instead please use `user_id` which exits in your database. The second reason why it fails is that you are using `post` column instead of `post_id` when creating comments. Please verify the data you have in your schema and fix this error. Your comment creation should look like this `Comment.create(post_id: first_post.id, user_id: first_user.id, text: 'Hi Tom!1')` instead of `Comment.create(post: first_post.id, author: first_user.id, text: 'Hi Tom!1')`. I added the correct data to be able to test the methods you have in your models. So please work on the seed data 😄.

Reply:
#### Thank you so much! @julie-ify  for the detailed explanation. Let me fix it. 👌